models/ed/R/met2model.ED2.R

```diff
@@ -49,13 +49,13 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
   ## check to see if the outfolder is defined, if not create directory for output
   dir.create(met_folder, recursive = TRUE, showWarnings = FALSE)
 
-  dm <- c(0, 32, 60, 91, 121, 152, 182, 213, 244, 274, 305, 335, 366)
-  dl <- c(0, 32, 61, 92, 122, 153, 183, 214, 245, 275, 306, 336, 367)
   month <- c("JAN", "FEB", "MAR", "APR", "MAY", "JUN", "JUL", "AUG", "SEP", "OCT", "NOV", "DEC")
-  mon_num <- c("01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12")
-
 
   day2mo <- function(year, day, leap_year) {
+    # DOY corresponding to start of each month without a leap year...
+    dm <- c(0, 32, 60, 91, 121, 152, 182, 213, 244, 274, 305, 335, 366)
+    # ...and with a leap year
+    dl <- c(0, 32, 61, 92, 122, 153, 183, 214, 245, 275, 306, 336, 367)
```

Comment:
These vectors aren't used anywhere else in this function. I moved them into this function to make it clearer what they were for.